reference/fleet/fleet-agent-release.md

```diff
@@ -0,0 +1,18 @@
+---
+navigation_title: "{{agent}} release"
+mapped_pages:
+  - https://www.elastic.co/guide/en/fleet/current/fleet-agent-release.html
+applies_to:
+  stack: ga
+products:
+  - id: fleet
+  - id: elastic-agent
+---
+
+# {{agent}} release process [fleet-agent-release]
+
+{{agent}} follows a release process aligned with the broader Elastic Stack release schedule. Changes and fixes are documented in the [release notes](../../release-notes/fleet-elastic-agent/index.md).
+
+## Independent Agent Release
+
+Independent Agent Release (IAR) is a fast-track release process for {{agent}} and Endpoint, designed to deliver critical fixes and updates independently of the full stack release. The IAR version format appends a build identifier `+build{yyyymmddhhmm}` to the semantic version it is based on, where `{yyyymmddhhmm}` is the release timestamp of the build.
```

Comment:
```suggestion
Independent Agent Release (IAR) is a hotfix release process for {{agent}} and Elastic Defend designed to deliver critical fixes and updates independently of the full stack release. The IAR release process is more conservative than a typical patch release, and only modifies the specific Elastic Agent components needed for a targeted fix. For example, an IAR hotfix release for Elastic Defend would only change the endpoint-security executable with the remaining executables being exactly those released in the previous patch. 

The IAR version format appends a build identifier `+build{yyyymmddhhmm}` to the semantic version it is based on, where `{yyyymmddhhmm}` is the release timestamp of the build.
```

Reply:
Added some other language to emphasize that we don't relax the quality level to obtain speed and these releases are more conservative in what they change.